doc/api_methods/AddList.md

```diff
@@ -0,0 +1,19 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array addList(array $list)`
+
+In MailPoet, subscribers are organized into lists. This method provides functionality for creating a new list.
+
+It returns a new list. See [Get Lists](GetLists.md) for a list data structure description.
+
+## Arguments
+### `$list` (required)
+
+An associative array which contains list data.
+
+| Property | Type | Description |
+| --- | --- | --- |
+| name (required) | string | A valid email address |
+| description (optional) | string\|null | A description of the list limited to 250 characters. |
```

Comment:
You mentioned the length here, but not anywhere else. I think we should specify the limit in all other files and properties... And even for the return types so that users know what they need to allocate.